### ISSUE TITLE
The initial parameters for a condensed phase

### ISSUE THREAD

**Issue by vaniamr** [patonlab/GoodVibes#45]
Hi,

I performed a solvation calculation in Gaussian and, I would like to know the following: when GoodVibes reads the output and identifies that the calculation was carried out in a condensed phase, what initial value does it assume for concentration 1 mol/L or 1 bar?

**Comment by luchini18** [comment]
Hi vaniamr, 
By default, GoodVibes will use a pressure of 1.0 atm for gas phase calculations, however this should be adjusted for solvated systems using the `-c` option. We recommend using `-c 1`, which will adjust to a standard state of 1 mol/L for your calculation output files, adding a correction to the entropy in the process.
Thanks for using the program! Let us know if you have further questions!

**Comment by vaniamr** [comment]
Hi, 
Yes! I used the -c option in my calculations. But my doubt is: The calculation that came out of the Gaussian already has corrections for the condensed phase, so the energy values present in the Gaussian output are already corrected for 1 mol / L. Having said that, GoodVibes, when I put the option -c, make a new correction in the values that are already corrected?
My situation is: I need some corrections to 0.0003 mol / L, so I used the option -c 0.0003. Hence what I want to know is whether a correction is made to the desired concentration directly, assuming that the value of the output is already corrected to 1 M. Or if the correction of the gas phase to the condensed phase is redone and then to the desired concentration, because if it is the last one I have phase change corrections being done twice.
